docs/UserGuide.md

```diff
@@ -14,23 +14,27 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `lingogo.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for your LingoGO!.
 
 1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * **`list`** : Lists all contacts.
+   * **`add`**`q/hi a/hola` : Adds a flashcard for `hi` to the LingoGO!.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`delete`**`3` : Deletes the flashcard with card number 3.
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+   * **`edit`**`1 q/hi a/hola` : Edit the flashcard with card number 1.
 
-   * **`clear`** : Deletes all contacts.
+   * **`flip`**`2` : Flips the flashcard with card number 2 to show or hide the answer.
+   
+   * **`test`**`1 a/good morning` : Tells the user whether the answer they entered for a flashcard is correct or wrong.
```

Comment:
already covered by #36. i think this can be removed here as its duplicated

Reply:
Agreed. For this section, I think it might be better if each person updates based on their assigned command so that the work done for each issue is clearer and there isn't any duplicate work done.

Reply:
`add` and `delete` has already been done and merged into master